docs/sop/UPDATE_VERSION.md

```diff
@@ -178,20 +178,20 @@ YII2_VERSION: "2.0.48"
 # make sure staging has been deployed successfully through the gitlab pipeline, eg. sd_gigadb
 # staging now should have the latest datasets, check the RSS feed in staging website
 # log in to the staging bastion
-% ssh -i "$aws.pem" centos@"$basion.ip"
-[centos@ip-10-xx-x-xx ~]$ pg_dump --version
+% ssh -i "$aws.pem" ec2-user@"$basion.ip"
```

Comment:
**suggestion**: Might as well fix typo in `basion.ip` if changes are being made to this file.